hbase-server/src/test/java/org/apache/hadoop/hbase/ipc/AbstractTestIPC.java

```diff
@@ -107,7 +108,8 @@ protected abstract RpcServer createRpcServer(final Server server, final String n
     final List<BlockingServiceAndInterface> services, final InetSocketAddress bindAddress,
     Configuration conf, RpcScheduler scheduler) throws IOException;
 
-  protected abstract AbstractRpcClient<?> createRpcClientNoCodec(Configuration conf);
+  protected abstract AbstractRpcClient<?> createRpcClientNoCodec(Configuration conf)
```

Comment:
After removing the throws declaration of NettyRpcClient, I think we do not need to change these lines then.

Reply:
Reverted.